why3/src/coma.rs

```diff
@@ -135,10 +136,6 @@ impl Defn {
 }
 
 impl Expr {
-    pub fn boxed(self) -> Box<Self> {
-        Box::new(self)
-    }
-
```

Comment:
Why don"t you like this?

That's a function I recently added, wimply because I find it easier to write and read `Constr(blah.boxed())` `Constr(Box::new(blah))`. One immediate advantage is the lower nesting of parentheses.